src/ngraph_builder.cc

```diff
@@ -345,13 +345,14 @@ Builder::TF_NGRAPH_CONST_MAP() {
 //  }
 static Status TranslateUnaryOp(
     const Node* op, const std::vector<const Tensor*>& static_input_map,
-    Builder::OpMap& ng_op_map,
+    Builder::OpMap& ng_op_map, Builder::OpMap& first_ng_op_map,
```

Comment:
Can we rename this to something like entry_nodes_ng_op_map and similary ng_op_map to exit_nodes_ng_op_map to make it more readable?

Reply:
I think `entry_nodes_ng_op_map` and `output_nodes_ng_op_map` make sense to me!

Reply:
outputs sound better than exit.

Reply:
In that case input_nodes_ng_op_map and output_nodes_ng_op_map makes more sense if you  don't want to change all the comments etc.

Reply:
input_nodes_ng_op_map might be confused with the actual input ng nodes to the node being translated, which is not what we are looking at. We are looking at the first nodes we create as part of this TF node's translation

Reply:
entry/exit , input/output make sense but we can do _entry_nodes_ng_op_map_ and _output_nodes_ng_op_map_